test/models/maintenance_tasks/task_data_test.rb

```diff
@@ -149,12 +149,13 @@ class TaskDataTest < ActiveSupport::TestCase
     end
 
     test "#parameter_names returns list of parameter names for Tasks supporting parameters" do
-      assert_equal ["post_ids"],
+      assert_equal ["post_ids", "content", "integer_attr"],
         TaskData.new("Maintenance::ParamsTask").parameter_names
     end
 
     test "#parameter_names returns empty list for deleted Tasks" do
-      assert_empty TaskData.new("Maintenance::DoesNotExist").parameter_names
+      names = TaskData.new("Maintenance::DoesNotExist").parameter_names
+      assert_equal [], names # to make it's not any empty object
```

Comment:
Nit: I think it's clear without the comment! 😄 
```suggestion
      assert_equal [], names
```